lib/src/markdown_processor.dart

```diff
@@ -156,6 +158,7 @@ RegExp get isConstructor => _constructorIndicationPattern;
 /// punctuation, spaces.
 ///
 /// The idea is to catch such cases and not produce warnings about the contents.
+/// TODO(jcollins-g): consider being more strict here.
```

Comment:
Typically a TODO comment would not be a part of the doc comment, but I'm not sure that we have a separate convention in this package. I see 100 instances of `/// TODO` in analyzer, so maybe we sometimes do include them?